README.md

```diff
@@ -559,3 +562,72 @@ section-1: value-1  # hasn't been changed
 section-2:
   key-21: value-21-new  # body was replaced
 ```
+
+### Stateboard instance
+
+Stateboard is a tarantool state provider for stateful failover.
+It's delivered with an application RPM/DEB package - if application contains
+`stateboard.init.lua` file in it's root, the application package contains
+`/etc/systemd/system/<appname>-stateboard.service` unit faile.
+
+It starts tarantool stateboard instance with an entry point
+`/usr/share/tarantool/<appname>/stateboard.init.lua`.
+This instance looks for it's configuration in `<appname>-stateboard` section
+across all files in `/etc/tarantool/conf.d` directory.
+
+This instance can be started using this role.
+To mark instance as a stateboard `stateboard` flag is used.
+
+Stateboard instance is started as `<app_name>-stateboard` systemd service.
+
+Stateboard can be configured using the `config` variable.
+This variable describes stateboard parameters that would be passed to its
+configuration.
+
+**Note** that `cartridge_defaults` params don't affect stateboard instance.
+
+#### Required config parameters
```

Comment:
What about lock_delay parameter? (it's optional)

Reply:
I sure that we need a link to stateboard doc here.